src/ray/common/bundle_spec.cc

```diff
@@ -109,12 +123,12 @@ std::string FormatPlacementGroupResource(const std::string &original_resource_na
 
 bool IsBundleIndex(const std::string &resource, const PlacementGroupID &group_id,
                    const int bundle_index) {
-  return resource.find("_group_" + std::to_string(bundle_index) + "_" + group_id.Hex()) !=
-         std::string::npos;
+  return resource.find(kGroupKeyword + std::to_string(bundle_index) + "_" +
```

Comment:
I think std::to_string here will benefit from this (https://github.com/ray-project/ray/pull/18538)

Are you seeing anything different after that PR merged?

Reply:
https://stackoverflow.com/questions/21694302/what-are-the-mechanics-of-short-string-optimization-in-libc

It'll benefit small string if it's less than 22 chars.